plugins/cloud-costs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.22
+FROM python:alpine3.22
```

Comment:
[opt] use `python:3-alpine3.22` instead, if python 4 is ever released, might break 😬